### DISCUSSION TITLE
How do you use the tvheadend add-on?

### DISCUSSION THREAD

**Discussion by 7wells** [dfigus/addon-tvheadend#135] [General]
Hello!

I'm curious about how you guys use the tvheadend add-on.

I have it on an RPi4 in my mobile car and watch TV via DVB-T2 on my tablet with Kodi (no recordings). It saves a lot on my mobile data plan if no WiFi is available (and DVB-T2 is available, of course 😉).

Thanks for sharing and for reviving this add-on! ❤️

**Comment by dfigus** [comment]
Hi,

thanks for starting this discussion! I was also thinking about starting a similar discussion, but I never managed to - got distracted by other stuff 😆.  I only use it to record from my FritzBox with SATIP DVB-C tuners and watch later via Kodi or Cast on our TV. The main use case is to schedule recordings when we are away, hence I also enabled the Ingress feature for his add-on so that I can use it via the HA remote UI. 

Previously scheduling recordings was possible on our Panasonic DVR/BD-Player via their Cloud-enabled App, but they switched it off last October and they don't offer a local API. That was my trigger to build / revive the add-on.

**Reply by 7wells** [reply]
Interesting, thanks! As I have a DVB-T2 settop box at home with a small but sufficient 2.5" HDD inside, I rely on that one for recordings. The tvheadend in my case is really rather for experimenting and watching on the road - well, not when I'm the driver. 😉
